static/styles/left_sidebar.css

```diff
@@ -164,6 +164,10 @@ li.show-more-topics {
     .inactive_stream {
         opacity: 0.5;
     }
+
+    .zulip-icon.zulip-icon-ellipsis-v-solid {
+        opacity: 0.7;
+    }
```

Comment:
Due to differences in the opacity of all three dots icon in the Stream section on the left sidebar compared to that on the global section which includes All messages, Recent Topics, Private messages, etc. There was inconsistency on the left-sidebar UI.

This commit applied Opacity on the stream section ellipsis icon only (The three dots) so they will be consistent with the overall ellipsis icon as that of the global section already has opacity attached to it.